newIDE/app/src/ObjectsList/ObjectSelector.js

```diff
@@ -18,6 +18,7 @@ type Props = {|
 
   onChoose?: string => void,
   onChange: string => void,
+  onRequestClose?: () => void,
```

Comment:
We probably want to remove the two lines added in this file because it's not useful